skills/cmux-customization/SKILL.md

```
@@ -0,0 +1,115 @@
+
+- cmux app preferences: use `cmux-settings` for `~/.config/cmux/cmux.json` settings such as appearance, sidebar, notifications, browser behavior, automation, and shortcuts.
+- Custom actions, workspace layouts, toolbar buttons, plus-button behavior, and Command Palette entries: edit `~/.config/cmux/cmux.json` globally or `.cmux/cmux.json` in the project.
+- Terminal rendering and terminal keybindings: use Ghostty config, usually `~/.config/ghostty/config`. This includes fonts, cursor style, copy-on-select, shell integration, themes, and terminal keybindings.
+- Project-specific behavior: prefer `.cmux/cmux.json` in the project so the customization travels with the repo.
+
+If a request can be handled by Ghostty config, say that and use Ghostty config instead of inventing cmux UI settings.
+
+## Workflow
+
+1. Inspect existing config before editing.
+   ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)